Watchman.Discord/Areas/Administration/Controllers/MutedUsersController.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Devscord.DiscordFramework.Framework.Architecture.Controllers;
+using Devscord.DiscordFramework.Middlewares.Contexts;
+using Devscord.DiscordFramework.Services;
+using Devscord.DiscordFramework.Services.Factories;
+using Watchman.Discord.Areas.Administration.BotCommands;
+using Watchman.Discord.Areas.Protection.Services;
+using Watchman.DomainModel.Messages;
+
+namespace Watchman.Discord.Areas.Administration.Controllers
+{
+    public class MutedUsersController : IController
+    {
+        private readonly MutingHelper _mutingHelper;
+        private readonly UsersService _usersService;
+        private readonly DirectMessagesService _directMessagesService;
+
+        public MutedUsersController(DirectMessagesService directMessagesService, MutingHelper mutingHelper, UsersService usersService)
+        {
+            this._directMessagesService = directMessagesService;
+            this._mutingHelper = mutingHelper;
+            this._usersService = usersService;
+        }
+
+        [AdminCommand]
+        public async Task MutedUsers(MutedUsersCommand mutedUsersCommand, Contexts contexts)
+        {
+            var mutedUsers = _usersService.GetUsersAsync(contexts.Server);
+            var (title, description, values) = await this.GetMuteEmbedMessage(mutedUsers, contexts.Server.Id);
+            if (values.Count == 0)
+            {
+                await _directMessagesService.TrySendMessage(contexts.User.Id, "Brak wyciszonych użytkowników!");
+                return;
+            }
+            await _directMessagesService.TrySendEmbedMessage(contexts.User.Id, title, description, values);
+        }
+
+        private async Task<(string title, string description, Dictionary<string, Dictionary<string, string>> values)> GetMuteEmbedMessage(IAsyncEnumerable<UserContext> mutedUsers, ulong serverId)
```

Comment:
jak tuple jest taki potężny (szeroki w zapisie), to lepiej już zrobić osobną klase w `Watchman.Discord.Areas.Administration.Models`
ze względu na czytelność